presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -911,6 +913,32 @@ public void testDateTimeOutputString()
         assertFunctionString("timestamp '2333-02-23 23:59:59.999 Asia/Tokyo'", TIMESTAMP_WITH_TIME_ZONE, "2333-02-23 23:59:59.999 Asia/Tokyo");
     }
 
+    @Test
+    public void testParseDuration()
+    {
+        // null duration
+        assertFunction("parse_duration(null)", INTERVAL_DAY_TIME, null);
```

Comment:
Remove this. It will never be called by the engine. We don't add dummy tests for null.